travis-docker-compose.yml

```diff
@@ -31,31 +31,25 @@ services:
       BROKER_URL: redis://redis:6379/4
       CELERY_RESULT_BACKEND: redis://redis:6379/4
     env_file: .env
+
+  web:
+    image: micromasters_python
```

Comment:
where does the `micromasters_python` image come from? is that tag automatically generated based on having the `python` container declared in the `docker-compose` file?